### PR TITLE
Feature cn 5 gl support

### PR DESCRIPTION
**Pull request by mandolyte**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-box3/content-validation-app/6)

### CODE REVIEW

**Review by PhotoNomad0 — APPROVED** [review]
> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-box3/content-validation-app/6)* status: :shipit: complete! all files reviewed, all discussions resolved

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.4.1",
     "typescript": "^4.0.2",
-    "uw-content-validation": "^0.8.9"
+    "uw-content-validation": "^0.8.10"
```
Comment (PhotoNomad0) [inline package.json:39]: need to update to latest version.

**Review by PhotoNomad0 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mandolyte) [reply package.json:39]: This is updated in the other/next pull request. So if both are done, then it is updated to 0.8.11.

**Review by PhotoNomad0 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-box3/content-validation-app/6)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**